samples/stock/demo/relative-rotation-graph/demo.js

```diff
@@ -0,0 +1,289 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'scatter',
+        height: '100%'
```

Comment:
```suggestion
        height: '100%',
        plotBorderColor: '#000000',
        plotBorderWidth: 1
```
There are some pixel inconsistencies with the border around the edge of the plot area. I think the best way to make that consistent is to
* Add a `plotBorderWidth` to the chart
* Remove the `strokeWidth` of the quadrant shapes
* Increase the z-index of the plot lines